next-env.d.ts

```diff
@@ -0,0 +1,5 @@
+/// <reference types="next" />
+/// <reference types="next/image-types/global" />
+
+// NOTE: This file should not be edited
+// see https://nextjs.org/docs/basic-features/typescript for more information.
```

Comment:
What is this for?

Reply:
This is TS type file, for the compiler to understand next.js types